integration_tests/tests/apply/about_the_person/equality_and_diversity/complete_disability_page.cy.ts

```diff
@@ -0,0 +1,113 @@
+//  Feature: Referrer completes disability question page
+//    So that I can complete the 'Equality questions' task
+//    As a referrer
+//    I want to answer questions on the disability page
+//
+//  Scenario: answer is enforced
+//    Given I'm on the 'Will answer equality questions?' task page
+//    When I try to continue without answering the question
+//    Then I see that an answer is required
+//
+//  Scenario: submit 'other' disability type
+//    Given I'm on the 'Will answer equality questions?' task page
+//    When I submit an 'other' disability type
+//    Then I return to the task list page
+//    And I see that the task has been completed
+//
+//  Scenario: return to equalities question using the back button
+//    Given I'm on the disability page
+//    When I use the back button
+//    Then I'm on the Equalities questions page
+//
+
+import Page from '../../../../pages/page'
+import TaskListPage from '../../../../pages/apply/taskListPage'
+import WillAnswerEqualityQuestionsPage from '../../../../pages/apply/willAnswerEqualityQuestionsPage'
+import DisabilityPage from '../../../../pages/apply/disabilityPage'
+import { personFactory, applicationFactory } from '../../../../../server/testutils/factories/index'
+
+context('Visit "About the person" section', () => {
+  const person = personFactory.build({ name: 'Roger Smith' })
+
+  beforeEach(function test() {
+    cy.task('reset')
+    cy.task('stubSignIn')
+    cy.task('stubAuthUser')
+
+    cy.fixture('applicationData.json').then(applicationData => {
+      const application = applicationFactory.build({
+        person,
+      })
+      application.data = applicationData
+      cy.wrap(application).as('application')
+      cy.wrap(application.data).as('applicationData')
+    })
+  })
+
+  beforeEach(function test() {
+    // And an application exists
+    // -------------------------
+    const application = applicationFactory.build({
+      id: 'abc123',
+      data: {
+        'funding-information': {
+          'funding-source': {},
+        },
+        'equality-and-diversity-monitoring': {
+          'will-answer-equality-questions': {},
+          disability: {},
+        },
+      },
+      person,
+    })
+    cy.task('stubApplicationGet', { application })
+    cy.task('stubApplicationUpdate', { application })
+
+    // Given I am logged in
+    //---------------------
+    cy.signIn()
+
+    // And I am on the disability question page
+    // --------------------------------
+    cy.visit('applications/abc123/tasks/equality-and-diversity-monitoring/pages/disability')
+    Page.verifyOnPage(DisabilityPage, this.application)
+  })
+
+  // Scenario: answer is enforced
+  // ----------------------------
+  it('enforces answer', function test() {
```

Comment:
do we want to keep the error handling test here, or did we decide to get rid as we're covering the behaviour in unit tests?
same with back button test below...seems quite repetitive...